test/integration/server.js

```diff
@@ -153,10 +153,12 @@ helpers.stubUtxos = function(server, wallet, amounts, cb) {
       return {
         txid: helpers.randomTXID(),
         vout: Math.floor(Math.random() * 10 + 1),
-        satoshis: helpers.toSatoshi(amount).toString(),
+        satoshis: helpers.toSatoshi(amount),
```

Comment:
IIRC the `toString()` is needed to simulate UTXOs as returned by Insight. Were you having issues with this?